src/webgpu/util/f32_interval.ts

```diff
@@ -115,25 +103,28 @@ export interface PointToInterval {
 }
 
 /** Operation used to implement a PointToInterval */
-interface PointToIntervalOp {
+export interface PointToIntervalOp {
   /** @returns acceptance interval for a function at point x */
-  impl: (x: number) => F32Interval;
+  impl: PointToInterval;
 
   /**
    * Calculates where in the domain defined by x the min/max extrema of impl
    * occur and returns a span of those points to be used as the domain instead.
    *
-   * If not defined the ends of the existing domain are assumed to be the
-   * extrema.
+   * If the result is not defined the ends of the existing domain are assumed to
+   * be the extrema.
    *
-   * This is only implemented for functions that meet all of the following
+   * This is only implemented for subclasses that meet all of the following
    * criteria:
    *   a) non-monotonic
    *   b) used in inherited accuracy calculations
    *   c) need to take in an interval for b)
    *      i.e. fooInterval takes in x: number | F32Interval, not x: number
    */
   extrema?: (x: F32Interval) => F32Interval;
+
+  /** @returns an additional interval that inputs need to be within. If not defined, they are only tested for being finite. */
```

Comment:
Comment is a little confusing given the interface.
What happens if the input is not in the `domain`? Who's doing the `tested` ?

Reply:
Better thank you. Still not sure you need this exposed though :)